ipatests/test_integration/test_nfs.py

```diff
@@ -106,6 +106,7 @@ def test_krb5_nfsd(self):
         basedir = "exports"
         exports = {
             "krbnfs": "*(sec=krb5p,rw)",
+            "krb5nfs": "*(sec=krb5,rw)",
```

Comment:
Are we sure we need another mountpoint?
I don't see why the problem would not exist with the existing krb5p mount.

Reply:
@fcami Thanks for the review.
It was added as it was in bugzilla description but as it can be reproducible  with krb5p also, so removed another mount-point.